doc/source/upgrading_guide.rst

```diff
@@ -94,8 +94,22 @@ If you only used the |MyRocks| storage engine in |Percona Server| |version.prev|
 
    $ apt-get install percona-server-rocksdb
 
+.. deprecated:: 8.0.16-7
 
-The installation script will *NOT* run automatically :command:`mysql_upgrade` as
+The :command:`mysql_upgrade` was deprecated as of |Percona Server|
+8.0.16-7. The functionality was moved to the `mysqld` binary which automatically
+runs the upgrade process, if needed. If you attempt to run `mysql_upgrade`,
+no operation happens and a message stating, "The mysql_upgrade client is not
```

Comment:
following:
If you attempt to run `mysql_upgrade`, no operation happens and a message stating, "The mysql_upgrade client is **not** deprecated. The actions executed by the upgrade client are **not** done by the server."

seems to me should be:
If you attempt to run `mysql_upgrade`, no operation happens and a message stating, "The mysql_upgrade client is **now** deprecated. The actions executed by the upgrade client are **now** done by the server."

This repeats several times on this page so it should be fixed in all places.